src/actors/symbolication.rs

```diff
@@ -1570,7 +1570,7 @@ impl SymbolicationActor {
         sources: Vec<SourceConfig>,
     ) -> Result<(SymbolicateStacktraces, MinidumpState), SymbolicationError> {
         let future = async move {
-            let sources = Arc::new(sources);
+            let sources: Arc<[SourceConfig]> = Arc::from(sources);
```

Comment:
do you really need the type annotation here?

Reply:
No, rust is smart enough. but otherwise the reader has to figure this out somehow by looking up what the type of the parameters are.  I don't mind too much though, I'm pretty sure @jan-auer will be of your opinion as well.

Reply:
> I'm pretty sure @jan-auer will be of your opinion as well.

Correct. 

Aside from that, please update tests, as now the types no longer match.

Reply:
lol, it **does** actually need this type.  sorry folks

Reply:
> Aside from that, please update tests, as now the types no longer match.

Not sure what you refer to with this comment.

Reply:
You resolved this in the meanwhile.